### PR TITLE
Spec/enhance gluster cluster sync logic

### PR DESCRIPTION
**Pull request by nnDarshan**

### CODE REVIEW

**Comment by r0h4n** [comment]
What is the difference between this spec vs https://github.com/Tendrl/specifications/pull/170

**Comment by nnDarshan** [comment]
@r0h4n its enhancement to #170, this has additional details about geo-replication related objects

File: specs/refactoring_of_node_monitoring_flows_into_node_agent.adoc [hunk 1]
```diff
@@ -0,0 +1,235 @@
+// vim: tw=79
```
Comment (r0h4n) [inline specs/refactoring_of_node_monitoring_flows_into_node_agent.adoc:1]: Remove this file, each spec PR should have one corresponding spec file

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: specs/enhance_gluster_cluster_sync_logic.adoc [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+cluster state based on peers status and volume status of the gluster trusted
+storage pool. Also the overall volume utilization should be calculated based on
+aggregations done for the participating bricks of the volumes.
+
+
+== Use Cases
+
+This addresses the synchronization of gluster cluster details like
+
+* Volume snapshot details
```
Comment (r0h4n) [inline specs/enhance_gluster_cluster_sync_logic.adoc:29]: If this is about geo-rep objects, why is Volume snapshot details and other redundant data (from #170 ) present here?

**Review by r0h4n — CHANGES_REQUESTED** [review]

**Comment by shtripat** [comment]
@r0h4n I have added the geo-rep relaped details in my spec https://github.com/Tendrl/specifications/pull/170 and updated already.

@nnDarshan you may close this PR I feel in that case.

**Comment by nnDarshan** [comment]
Closing this as @shtripat has added the details of this PR in the original PR

**Pull request closed without merging**